Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person.hs

```diff
@@ -199,10 +199,10 @@ findAllDriversByIdsFirstNameAsc merchantId driverIds = do
       info <- HashMap.lookup driverId' driverInfoHashMap
       Just $ FullDriver person location info vehicle
 
-getOnRideStuckDriverIds :: (MonadFlow m, Log m) => m [DriverInformation]
-getOnRideStuckDriverIds = do
+getOnRideStuckDriverIds :: (MonadFlow m, Log m) => UTCTime -> m [DriverInformation]
+getOnRideStuckDriverIds dbSyncInterVal = do
   driverIds <- findAllWithDb [Se.Is BeamR.status $ Se.In [Ride.INPROGRESS, Ride.NEW]] <&> (Ride.driverId <$>)
-  findAllWithKV [Se.And [Se.Is BeamDI.onRide $ Se.Eq True, Se.Is BeamDI.driverId $ Se.Not $ Se.In (getId <$> driverIds)]]
```

Comment:
Why you changed this to Db from KV?

Reply:
because to sync changes, because above driver Id we are getting from db only , so i think it is better t o get them from there only.